### PR TITLE
update viz and settings buttons

### PR DESCRIPTION
**Pull request by uladzimirdev**
[Epic](https://github.com/metabase/metabase/issues/47911)

update visualization settings button text

https://github.com/user-attachments/assets/d2c1bee3-94ea-4254-b617-c2a6139f7735

### CODE REVIEW

File: frontend/src/metabase/query_builder/components/view/ViewButton.tsx [hunk 1]
```diff
@@ -31,6 +32,10 @@ const ViewButton = styled(Button)<Props>`
   > .Icon {
     opacity: 0.6;
   }
+
+  .Icon + div {
```
Comment (ranquild) [inline frontend/src/metabase/query_builder/components/view/ViewButton.tsx:36]: to not target some unrelated diff maybe:
```
.Icon:not(:last-child) {
  margin-right: 0.25rem;
}
```

**Review by ranquild — APPROVED** [review]

File: frontend/src/metabase/query_builder/components/view/ViewButton.tsx (hunk 1 above)
Comment (uladzimirdev) [reply frontend/src/metabase/query_builder/components/view/ViewButton.tsx:36]: we need to override styles of the sibling, that's why I used `.Icon + div`
